tools/package-lock.json

```diff
@@ -1,7 +1,7408 @@
 {
   "name": "app-dev-tools",
+  "lockfileVersion": 2,
```

Comment:
@theswerd - did we discuss upgrading to lockfileVersion 2 before? Any other implications to worry about?

Reply:
@theswerd - please comment if there's any issues with lockfileversion 2